packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,135 @@
+      <SfLink link="/">
+        <SfImage v-if="logo" :src="logo" :alt="title" class="sf-header__logo" />
+        <h1 v-else class="sf-header__title">{{ title }}</h1>
+      </SfLink>
+    </slot>
+    <div class="sf-header__aside">
+      <!-- @slot Use this slot for language or currency selector -->
+      <slot name="aside" />
+    </div>
+    <div class="sf-header__actions">
+      <nav class="sf-header__navigation">
+        <slot name="navigation">
+          <div v-if="!isVisibleOnMobile" class="sf-header__menu">
+            <SfButton
+              v-for="(item, i) in menuItems"
+              :key="i"
+              class="sf-header__menu-item sf-header__menu-item--desktop sf-button--pure"
+              v-on="$listeners"
+              @mouseenter="$emit('mouseenter:button', item)"
+              @mouseleave="$emit('mouseleave:button', '')"
+              @click="$emit('click:button', item)"
+            >
+              {{ item }}
+            </SfButton>
+          </div>
+          <SfSidebar
+            :visible="isVisibleOnMobile && openSidebar"
+            :persistent="true"
+            :overlay="false"
+            class="sf-header__menu--sidebar"
+            @close="$emit('close')"
+          >
+            <SfMenuItem
+              v-for="(item, i) in menuItems"
+              :key="i"
+              :label="item"
+              class="sf-header__menu-item sf-header__menu-item--mobile"
+              @click="$emit('click:button', item)"
             />
-            <h1 v-else class="sf-header__title">{{ title }}</h1>
-          </SfLink>
+          </SfSidebar>
+          <div v-if="openContent" class="sf-header__menu-content">
+            <template v-for="item in menuItems">
+              <slot :name="item" />
+            </template>
+          </div>
         </slot>
-        <div class="sf-header__aside">
-          <!-- @slot Use this slot for language or currency selector -->
-          <slot name="aside" />
-        </div>
-        <div class="sf-header__actions">
-          <nav
-            class="sf-header__navigation"
-            :class="{ 'is-visible': isNavVisible }"
+      </nav>
+      <!--@slot Use this slot to replace default search bar-->
+      <slot
+        v-if="isVisible"
+        name="search"
+        v-bind="{ searchValue, searchPlaceholder }"
+      >
+        <SfSearchBar
+          :value="searchValue"
+          :placeholder="searchPlaceholder"
+          aria-label="Search"
+          class="sf-header__search"
+          @input="$emit('change:search', $event)"
+          @enter="$emit('enter:search', $event)"
+        />
+      </slot>
+      <!--@slot Use this slot to replace default header icons with custom content-->
+      <slot
+        name="header-icons"
+        v-bind="{
+          activeIcon,
+          cartIcon,
+          wishlistIcon,
+          accountIcon,
+        }"
+      >
+        <div v-if="isVisible" class="sf-header__icons">
+          <SfButton
```

Comment:
I think we can use a `v-for` here to loop over the array of menu items

Reply:
I created separate prop for array of icons.